httpie/output/streams.py

```
@@ -0,0 +1,293 @@
+
+class EncodedStream(BaseStream):
+    """Encoded HTTP message stream.
+
+    The message bytes are converted to an encoding suitable for
+    `self.env.stdout`. Unicode errors are replaced and binary data
+    is suppressed. The body is always streamed by line.
+
+    """
+    CHUNK_SIZE = 1
+
+    def __init__(self, env=Environment(), **kwargs):
```

Ruff (0.8.2)

Do not perform function call `Environment` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)